_pytest/test_training_data.py

```diff
@@ -163,3 +164,15 @@ def test_nonascii_entities():
         assert entity["start"] == 19
         assert entity["end"] == 27
         assert entity["entity"] == "description"
+
+
+@pytest.mark.parametrize("data_file", [
+    "data/examples/wit/demo-flights.json",
+    "data/examples/luis/demo-restaurants.json",
+    "data/examples/api/"])
+def test_training_data_conversion(tmpdir, data_file):
+    out_path = tmpdir.join("rasa_nlu_data.json")
+    convert_training_data(data_file, out_path.strpath)
+    td = load_data(out_path.strpath)
+    assert td.entity_examples != []
+    assert td.intent_examples != []
```

Comment:
I think it would be helpful to keep a copy of the translated file in the repo and check against it. Otherwise it might be easy to forget toexplicitly check the output when changing the data conversion